app/packages/components/src/components/StatusButton/StatusButton.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import {
+  Button,
+  ButtonGroup,
+  Typography,
+  Box,
+  Tooltip,
+  ButtonProps,
+} from "@mui/material";
+import SettingsIcon from "@mui/icons-material/Settings";
+import CircleIcon from "@mui/icons-material/FiberManualRecord";
+import { useTheme } from "../";
+import _ from "lodash";
+
+const ButtonStylesOverrides: ButtonProps["sx"] = {
+  color: (theme) => theme.palette.text.secondary,
+  border: "1px solid",
+  borderColor: (theme) => `${theme.palette.divider} !important`,
+  textTransform: "none",
+  fontSize: "1rem",
+};
+
+type Severity = "info" | "success" | "warning" | "error";
```

Comment:
probably need to add `disabled` and `enabled` here too

Reply:
yep! thanks